src/Sulu/Component/SmartContent/Orm/DataProviderRepositoryTrait.php

```diff
@@ -315,4 +325,37 @@ protected function appendDatasource($datasource, $includeSubFolders, QueryBuilde
         // empty implementation can be overwritten by repository
         return [];
     }
+
+    /**
+     * Extension point to append order.
+     *
+     * @param array $sortBy
+     * @param string $sortMethod
+     * @param QueryBuilder $queryBuilder
+     * @param string $alias
+     * @param string $locale
+     *
+     * @return array parameters for query
+     */
+    protected function appendSortBy($sortBy, $sortMethod, QueryBuilder $queryBuilder, $alias, $locale)
+    {
+        foreach ($sortBy as $column) {
```

Comment:
@wachterjohannes That line is breaking other data providers, e.g. the `ContactDataProvider`: 
![warning invalid argument supplied for foreach 500 internal server error](https://cloud.githubusercontent.com/assets/405874/21254942/21f10244-c369-11e6-8a62-efa213b86def.png)